playbooks/byo/openshift-node/config.yml

```diff
@@ -0,0 +1,11 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
+
+- include: ../../common/openshift-cluster/std_include.yml
+
+- include: ../../common/openshift-node/config.yml
+  vars:
+    openshift_cluster_id: "{{ cluster_id | default('default') }}"
+    openshift_debug_level: "{{ debug_level | default(2) }}"
+    openshift_master_etcd_hosts: "{{ groups.etcd | default([]) }}"
+    openshift_master_etcd_port: 2379
```

Comment:
I wonder if we need all these vars? I suspect we could remove `openshift_cluster_id` at least.

Reply:
I was thinking the same.  I think most of that was related to the `bin/cluster` stuff.  I'm going to drop the vars and test an install.